tests/models/test_popular.py

```diff
@@ -212,3 +219,67 @@ def test_i2i(
     def test_second_fit_refits_model(self, dataset: Dataset) -> None:
         model = PopularModel()
         assert_second_fit_refits_model(model, dataset)
+
+
+class TestPopularModelConfiguration:
+
+    def test_from_config(self) -> None:
+        config = {
+            "popularity": "n_interactions",
+            "period": timedelta(days=7),
```

Comment:
please test different cases (use parametrize):
- period is None, period is given as an object, period is given as a serialised value
- same for begin_from

Same for get_config

Reply:
Same for popular in category for all complex parameters

Reply:
Different test cases were added.